moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -68,9 +70,6 @@ const ComponentForm = ({
   onSave,
   initialFormValues = null,
 }) => {
-  const doesInitialValueHaveSubcomponents =
```

Comment:
I changed this up to just watch if the currently selected component type has subcomponents since we are switching between component types now.